### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Aorta3698**
Please review my changes. 

In particular, this pull request addresses

#1 
#2 
#3 
#4

### CODE REVIEW

File: src/Login.js [hunk 1]
```diff
@@ -1,55 +1,49 @@
-import React, {useState} from 'react';
+import React, { useState } from "react";
```
Comment (anthony-yeo) [inline src/Login.js:1]: Single Quote Format?

**Review by anthony-yeo — COMMENTED** [review]
Everything looks good, I noticed a lot file were deleted. Were those files refactored into different files?
Also is everything under the __tests__ directory

**Comment by Aorta3698** [comment]
> 

I think a lot of files are from when @RandallCaler copied over our old react assignment and most of them remained unused. I removed those that don't appear to be in use. 

I also refactored some of them, so that it makes more sense. Such as the `App.js` file should not have been the first (welcome) page etc.

**Comment by anthony-yeo** [comment]
Ok sounds good, just wondering. Everything else looks good to me though!

File: src/Login.js (hunk 1 above)
Comment (Aorta3698) [reply src/Login.js:1]: I guess I can tell prettier to follow single quote format. I can look into it later. 

All the test cases are under `__tests__` directory. 

https://github.com/anarchaworld/307-Gather-Frontend/pull/5#issuecomment-1455206266

**Review by anthony-yeo — APPROVED** [review]
Good work!

File: src/App.js [hunk 2]
```diff
@@ -1,136 +1,16 @@
...
-            element={
-              <CreateAccount
-              />
-            }
-          />
-          <Route path="/login" element={<Login />} />
-        </Routes>
-      </BrowserRouter>
-    </div>
+    <Routes>
```
Comment (abentler) [inline src/App.js:9]: App.js looks readable and clean, love it

**Review by abentler — APPROVED** [review]
Looks very good. I can see why this took so long.

**Review by RandallCaler — APPROVED** [review]
Looks great! Much cleaner than before

**Pull request merged**